doc/modules/impute.rst

```diff
@@ -58,8 +58,8 @@ in the matrix. This format is thus suitable when there are many more missing
 values than observed values.
 
 The :class:`SimpleImputer` class also supports categorical data represented as
-string values or pandas categoricals when using the ``most_frequent`` or
-``constant`` strategy::
+string values or pandas categoricals when using the `most_frequent` or
```

Comment:
I think Joel meant single normal quotes, to make clear that it is a string 'most_frequent'

(or for me the alternative is also both: ``` ``'most_frequent'``  ```)